executor/src/witgen/jit/identity_queue.rs

```diff
@@ -132,20 +157,25 @@ where
 
 #[derive(Clone)]
 pub enum QueueItem<'a, T: FieldElement> {
+    Equation {
```

Comment:
This seems to be used for polynomial identities now. AFAICS, storing a polynomial identity in `QueueItem::Identity` will fail now.

So maybe we can change the `Identity` variant to `BusSend`? And drop support for connect identities entirely?

Reply:
Yes, we can do that in a next step.